src/components/pages/CommunityPage/shared/ContentCard/ContentCard.tsx

```diff
@@ -6,12 +6,13 @@ type ActionButtonProps = {
   id: string;
   text: string;
   href: string;
-  type: "primary" | "normal" | "link";
+  type: "primary" | "normal" | "link" | "transparentwhite" | "transparentblack";
```

Comment:
Google told me that Spanish uses capital letters much less often than English 😄

`transparentwhite` => `transparentWhite`
`transparentblack` => `transparentBlack`